arns/arns-token-1.md

```diff
@@ -72,6 +79,8 @@ The **ARNS-TOKEN-1** specification leverages the general AO Token and Subledger
 -- ARNS-TOKEN-1 Objects
 Name = Name or "Arweave Name Token" -- optional
 Ticker = Ticker or "ANT" -- optional
+Description = Description or "This is an Arweave Name Token." -- optional
+Keywords = Keywords or {} -- optional
```

Comment:
Odd to say its optional when you set a default value. I associate "optional" with being allow to not exist, eg nullable. If this is not the case (and I doubt it is) then we should remove these comments.

If it is in fact optional, we can say "<var> or nil"

Reply:
Yeah i guess to your point - they should all be mandatory.  Removed "optional"